src/Umbraco.Infrastructure/PropertyEditors/ImageCropperPropertyValueEditor.cs

```diff
@@ -188,13 +186,13 @@ internal class ImageCropperPropertyValueEditor : DataValueEditor // TODO: core v
         }
 
         // update json and return
-        if (editorJson == null)
+        if (editorImageCropperValue == null)
         {
             return null;
         }
 
-        editorJson["src"] = filepath == null ? string.Empty : _mediaFileManager.FileSystem.GetUrl(filepath);
-        return editorJson.ToString(Formatting.None);
+        editorImageCropperValue.Src = filepath == null ? string.Empty : _mediaFileManager.FileSystem.GetUrl(filepath);
```

Comment:
More pattern matching 😁 
```suggestion
        editorImageCropperValue.Src = filepath is null ? string.Empty : _mediaFileManager.FileSystem.GetUrl(filepath);
```